// [[Reason: Implicit undefined not fixed by code mod yet]] ////

//// [tests/cases/compiler/defaultParameterAddsUndefinedWithStrictNullChecks.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,12 +1,12 @@
 
 
 //// [defaultParameterAddsUndefinedWithStrictNullChecks.d.ts]
 declare function f(addUndefined1?: string, addUndefined2?: number): number;
-declare function g(addUndefined: string | undefined, addDefined: number): number;
+declare function g(addUndefined: string, addDefined: number): number;
 declare let total: number;
-declare function foo1(x: string | undefined, b: number): void;
-declare function foo2(x: string | undefined, b: number): void;
+declare function foo1(x: string, b: number): void;
+declare function foo2(x: string, b: number): void;
 declare function foo3(x: string | undefined, b: number): void;
 declare function foo4(x: string | undefined, b: number): void;
 type OptionalNullableString = string | null | undefined;
 declare function allowsNull(val?: OptionalNullableString): void;
